website/app/docs/concepts/templates/page.mdx

```diff
@@ -0,0 +1,146 @@
+# Kardinal Templates
```

Comment:
Let's call these "Flow Templates" instead of "Kardinal Templates" throughout this - it's going to be a more direct and intuitive label for the concept (cc @tc87 )